README.md

```diff
@@ -56,10 +56,10 @@ development mode:
 
 ```console
 # Linux:
-alias dex="stack exec dex --"
+alias dex="stack exec dex -- --lib-path lib"
```

Comment:
I wonder if passing `--lib-path` is now a required argument to commands like `dex script` and `dex web`? Or is it optional?

Reply:
It's optional, but it's usually what you want in "developer mode" so that you can make changes to dex libraries without having to run `make install` again to copy them to wherever stack likes to put its data files.

Reply:
Interesting, that makes sense. What happens with non-developer mode (optimized & installed production mode)?